src/pages/billing/NewBillingProjectWizard/StepWizard/Step.test.ts

```diff
@@ -22,9 +22,9 @@ describe('Step', () => {
   })
 
   it('sets aria-current to step when active', () => {
-    const renderResult = render(h(div, {}, [
-      Step({ isActive: true, children: [div({ children: ['activeStepChild'] })] }),
-      Step({ isActive: false, children: [div({ children: ['inactiveStepChild'] })] })
+    const renderResult = render(div([
```

Comment:
RHH functions such as `div` should be called directly instead of passed to `h`. Also, there's no need to pass an empty props object. It can be omitted.